cla_public/static-src/javascripts/modules/find-legal-adviser.js

```diff
@@ -307,3 +307,18 @@
     }
   };
 
+  function setMapHeight(setHeight) {
+    if ($(window).width() > 640) {
```

Comment:
Would I be right in guessing that this is the breakpoint for switching from side-by-side to one-above-the-other modes? Wondering if there's a more direct way of expressing this, that's less susceptible to inadvertent changes in the future. Also worth double-checking that that point is > 640 rather than >=, just for nasty single-pixel edge effect bug avoidance.

Reply:
Yes and no.  The 640 value is still hard-coded into the JS file, but I have merged the two by declaring `var wideScreen = 641;` at the top and then using it in both places (changing > to >= accordingly).  

It is still susceptible to changes in the break point, but just one loose end to tie up.  

Off the back of this, I have also done a check for the 460 value, by getting the original height of the map on load, before the sizing function is run.  This will mean that a change in the SCSS variable `$map-height` will be picked up by the JS.